### PR TITLE
Add Vectorian Giotto

### PR DESCRIPTION
**Pull request by marjuanm**
Vectorian Giotto is a free Adobe Flash clone, it's currently discontinued (the website no longer exists), however it's still possible to visit their website on the Wayback machine and download the installer.

This works without problems from Windows XP to Windows 10, I have found that Vectorian Giotto works without problems in ReactOS from 0.4.15-dev-6739 onwards, as you can see in this forum thread https://reactos.org/forum/viewtopic.php?p=147907#p147907

### CODE REVIEW

File: vectoriangiotto.txt [hunk 1]
```diff
@@ -0,0 +1,10 @@
+[Section]
+Name = Vectorian Giotto
+Version = 3.0.0
+License = Freeware
+Description = Vectorian Giotto is a free Adobe Flash clone application that allows you to create animations in swf format in a similar way to Adobe's commercial application.
+Category = 3
+URLSite = http://vectorian.com/ (discontinued)
```
Comment (binarymaster) [inline vectoriangiotto.txt:7]: RAPPS expects this to be a valid link without additional text.
```suggestion
URLSite = https://web.archive.org/web/20120505094833/http://vectorian.com/
```

File: vectoriangiotto.txt [hunk 2]
```diff
@@ -0,0 +1,10 @@
+[Section]
+Name = Vectorian Giotto
+Version = 3.0.0
+License = Freeware
```
Comment (binarymaster) [inline vectoriangiotto.txt:4]:
```suggestion
License = Freeware
LicenseType = 2
```

**Review by binarymaster — COMMENTED** [review]
You might want to add the icon and screenshots. Also for the future, no need to create a Jira ticket if you created a rapps-db PR.

**Comment by marjuanm** [comment]
ok, thanksvery much for your reply, I attach app icon and screenshots.
giotto01
giotto02
giotto03
vectoriangiotto

**Comment by binarymaster** [comment]
Is this exclamation mark the part of software or its component?

image

**Comment by marjuanm** [comment]
> Is this exclamation mark the part of software or its component?
> 
> image

No, sorry, this icon is added by VirtualBox only (recent version set a sidebar and is activated with the icon on screen).

**Comment by binarymaster** [comment]
> No, sorry, this icon is added by VirtualBox only (recent version set a sidebar and is activated with the icon on screen).

I have no such problem with VirtualBox here. Use the HostKey+E key combination to save screenshots from VM directly.

**Comment by marjuanm** [comment]
> > No, sorry, this icon is added by VirtualBox only (recent version set a sidebar and is activated with the icon on screen).
> 
> I have no such problem with VirtualBox here. Use the HostKey+E key combination to save screenshots from VM directly.

Ok, I will upload the screenshots that way (or another that does not have that problem), I have also uploaded the icon, sorry, I am still a newbie to Github and maybe I will make several mistakes (such as uploading the file several times or uploading it wrong , please bear with me a little).

**Comment by binarymaster** [comment]
> I am still a newbie to Github and maybe I will make several mistakes (such as uploading the file several times or uploading it wrong , please bear with me a little).

It's no problem, I'm just pointing out what needs to be addressed to get it merged. Thanks.

**Review by binarymaster — APPROVED** [review]

File: vectoriangiotto.txt [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+LicenseType = 2
```
Comment (binarymaster) [inline vectoriangiotto.txt:5]: To avoid doubling "Freeware (Freeware)":
```suggestion
LicenseType = 2
```

**Pull request merged**